lib/at_ex/gateway/Payments/topup_stash.ex

```diff
@@ -1,3 +1,66 @@
 defmodule AtEx.Gateway.Payments.TopupStash do
   @moduledoc false
+  import AtEx.Util
+
+  @live_url "https://payments.africastalking.com"
+  @sandbox_url "https://payments.sandbox.africastalking.com"
+
+  use AtEx.Gateway.Base, url: get_url(@live_url, @sandbox_url), type: "json"
+
+  @doc """
```

Comment:
@kamalogudah kindly update the docs here accordingly, as per the stash
Then I suggest in this, let us add the `productName` as an input in the map as it might be any of the existing App names from the already existing key names.